pylint/plugins/hass_enforce_type_hints.py

```diff
@@ -101,6 +101,7 @@ class ClassTypeHintMatch:
     "aiohttp_server": "Callable[[], TestServer]",
     "area_registry": "AreaRegistry",
     "async_setup_recorder_instance": "RecorderInstanceGenerator",
+    "calls": "list[ServiceCall]",
```

Comment:
This is not a "standard" fixture, but it is very common in tests, with about 80 instances in various components.
I think it's worth adding here